src/Components/WebView/test/E2ETest/BasicBlazorHybridTest.cs

```diff
@@ -0,0 +1,191 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using Microsoft.AspNetCore.Components.WebView.Photino;
+using Microsoft.Extensions.DependencyInjection;
+using PhotinoNET;
+
+namespace Microsoft.AspNetCore.Components.WebViewE2E.Test;
+
+public class BasicBlazorHybridTest
+{
+    private string _latestControlDivValue;
+
+    public void Run()
+    {
+        // Note: This test produces *a lot* of debug output to aid when debugging failures. The only output
+        // that is necessary for the functioning of this test is the "Test passed" at the end of this method.
+
+        Console.WriteLine($"Current directory: {Environment.CurrentDirectory}");
+        Console.WriteLine($"Current assembly: {typeof(Program).Assembly.Location}");
+        var thisProgramDir = Path.GetDirectoryName(typeof(Program).Assembly.Location);
+
+        // Add correct runtime sub-folder to PATH to ensure native files are discovered (this is supposed to happen automatically, but somehow it doesn't...)
```

Comment:
It might be due to the ASP.NET Core repo being configured to emit `bin`/`obj` build output to `$(RepoRoot)/artifacts/...` instead of into a subdir within each project.

In `Microsoft.AspNetCore.Components.E2ETests.csproj`, we explicitly reset the `BaseOutputPath` and `OutputPath` MSBuild properties to override that and get the default `bin`/`obj` placement back, since those tests need it for some reason. Possibly doing that in this new project would make it work too.

However, since we don't really need to prove that Photino itself works in any particular way (we only need to show our libraries work when Photino is working), your workaround seems entirely fine to me as-is.